i3: add common page (#4960)
tldr-pages/tldr: pages/linux/i3.md

# i3

> A dynamic tiling window manager.
> More information:<https://i3wm.org/docs/userguide.html>.

> - Start i3 (Note you must not have a window manager open when using this command.):

Reviewer: Generally we try to avoid the 2nd person (e.g. `you`, `your`, etc) in tldr-pages :slightly_smiling_face:
```suggestion
- Start i3 (Note that a pre-existing window manager must not be open when this command is run.):
```

`i3`

- Open a new terminal window:

> `Super+Return`

Reviewer: Maybe we could have spaces either side of the `+` here? I think it would make it easier to read at a glance, and we do this on other pages if I recall correctly.

Author: Ah didn't notice that my bad. I'll edit the rest of the document to follow that!

# i3

> A dynamic tiling window manager.
> More information:<https://i3wm.org/docs/userguide.html>.

- Start i3:

Note you must not have a window manager open when using this command.

`i3`

- Open a new terminal window:

`Super+Return`

- Create a new workspace:

`Super+Shift+{{number}}`

- Switch to workspace {{number}}:

`Super+{{number}}`

- Toggle fullscreen mode:

`Super+f`

- Kill current window:

`Super+Shift+q`

- Open new window horizontally:

`Super+h`

- Open new window vertically:

`Super+v`

> - Open application:
> (type out application name after executing command)

Reviewer 2: ```suggestion
- Open application (type out application name after executing command):

```
